src/lib/types/bar-chart.ts

```diff
@@ -0,0 +1,26 @@
+export enum BorderSkips {
+  start = 'start',
+  end = 'end',
+  middle = 'middle',
+  bottom = 'bottom',
+  left = 'left',
+  top = 'top',
+  right = 'right',
+  false = 'none',
+  true = 'all'
+}
```

Comment:
```suggestion
export type BorderSkips =
  | 'start'
  | 'end'
  | 'middle'
  | 'bottom'
  | 'left'
  | 'top'
  | 'right'
  | 'none'
  | 'all';
```

Reply:
I think union types are much better than enums in TypeScript, this will need changes elsewhere if you accept it

Reply:
https://github.com/svelte-hackathon/data-visualization/pull/2/commits/6b00423c696104e06cd5df1776cc5d53db3f65db 
https://github.com/svelte-hackathon/data-visualization/pull/2/commits/e25f7a4a7547701a38efe455212522f7a29c12f5
(Additionally corrects an error of not properly including false as permitted: https://www.chartjs.org/docs/latest/configuration/elements.html#bar-configuration)